src/main/java/Itstime/planear/schedule/Domain/Schedule.java

```diff
@@ -0,0 +1,38 @@
+package Itstime.planear.schedule.Domain;
+import java.time.LocalDateTime;
+
+import Itstime.planear.common.BaseEntity;
+import Itstime.planear.member.domain.Member;
+import jakarta.persistence.*;
+import lombok.Getter;
+
+@Entity
+@Getter
+@Table(name = "schedule")
+public class Schedule extends BaseEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private String title;
+
+    @Column
+    private LocalDateTime startDate;
+    private LocalDateTime endDate;
```

Comment:
column 다 넣어주면 좋겠어!
이런거 빼먹으면 나중에 네이밍 컨벤션 바뀌면 고생한다구
```suggestion
@Column(name="endDate")
    private LocalDateTime endDate;
```